WordPress/gutenberg
# Comments Pagination: Add border and spacing support

## What?
Add border block support to the comments pagination block.
Part of https://github.com/WordPress/gutenberg/issues/43247
Part of https://github.com/WordPress/gutenberg/issues/43243

## Why?
The comments pagination block is missing border support.

## How?
Adds the border block support in block.json

## Testing Instructions

1. Go to the Global Styles setting ( under appearance > editor > styles > edit styles > blocks )
2. Make sure that the comments pagination block's border and spacing are configurable via Global Styles
3. Verify that Global Styles are applied correctly in the editor and frontend
4. Edit the template/page, Add a comments pagination block and Apply the border and spacing styles
5. Verify that block styles take precedence over global styles
6. Verify that block borders and spacing display correctly in both the editor and frontend
